### DISCUSSION TITLE
How to highlight current word not underscore

### DISCUSSION THREAD

**Discussion by kaplax** [LunarVim/LunarVim#3325] [Q&A]
![image](https://user-images.githubusercontent.com/25189091/197394923-8c158e7a-4576-4951-b285-06a2661adff4.png)

### In this version
![image](https://user-images.githubusercontent.com/25189091/197394983-380ae274-8dd7-4a5c-ab0a-ae440a061040.png)

### I hope it like this
![image](https://user-images.githubusercontent.com/25189091/197395067-75fa9569-29c1-442e-8eeb-e5add6768bc2.png)

**Comment by LostInTheLogs** [comment]
This is set by your colorscheme. You can override it like this:
```lua
lvim.autocommands = {
  {
    { "ColorScheme" },
    {
      pattern = "*",
      callback = function()
        local col = "#383735" -- change it to your liking
    
        vim.api.nvim_set_hl(0, "IlluminatedWordRead", { bg = col, underline = false, bold = true })
        vim.api.nvim_set_hl(0, "IlluminatedWordText", { bg = col, underline = false, bold = true })
        vim.api.nvim_set_hl(0, "IlluminatedWordWrite", { bg = col, underline = false, bold = true })
      end,   
     }
  },
  -- other autocmds
}

```
